Themis/Views/Assessment/CorrectionSidebar/FeedbackListView.swift

```diff
@@ -42,6 +44,18 @@ struct FeedbackListView: View {
         }
     }
     
+    @ViewBuilder
+    private var athenaHintBox: some View {
```

Comment:
I would rename that variable to automaticFeedbackHintBox or something. Not everybody knows what Athena is.
```suggestion
    private var automaticFeedbackHintBox: some View {
```